### PR TITLE
ImagePlugin: Use original image option

### PR DESCRIPTION
**Pull request by cyberstar**
Allows user to choose if plugin should use thumbnail or the original image.

### CODE REVIEW

File: aldryn_bootstrap3/models.py [hunk 1]
```diff
@@ -234,6 +234,15 @@ class Boostrap3ImagePlugin(CMSPlugin):
         blank=True,
         default='',
     )
+    use_original_image = models.BooleanField(
+        _("use original image"),
+        blank=True,
+        default=False,
+        help_text=_(
+            "use original image in templates instead of generated "
+            "thumbnails."
```
Comment (stefanfoulis) [inline aldryn_bootstrap3/models.py]: ``use the original full-resolution image. no resizing.``

**Comment by stefanfoulis** [comment]
In the plugin form we should use JS to hide fields that are no longer relevant if ``use_original_image`` is selected.

Off the top of my head I think these fields should be hidden. Maybe there are more:
* ``override_width``
* ``override_height``
* ``aspect_ratio``

**Comment by vxsx** [comment]
@stefanfoulis added that functionality

**Comment by stefanfoulis** [comment]
:+1: nice :smile:

**Pull request merged**